templates/core/index.html

```diff
@@ -516,7 +516,7 @@ <h3 class="p-heading--4"><a href="/engage/iot-disk-encryption" onclick="dataLaye
           <p class="p-heading-icon__title">Webinar</p>
         </div>
       </div>
-      <h3 class="p-heading--4"><a href="/engage/intro-to-ubuntu-core22-webinar" onclick="dataLayer.push({' event' : 'GAEvent', 'eventCategory' : 'product page' , 'eventAction' : 'core - left' , 'eventLabel' : 'Webinar - Introuction to Ubuntu Core 20 ' , 'eventValue' : '1' });">Introduction to Ubuntu Core 22&nbsp;&rsaquo;</a></h3>
+      <h3 class="p-heading--4"><a href="/engage/intro-to-ubuntu-core22-webinar#:~:text=With%204%20releases%20behind%2C%20Ubuntu,edge%20devices%20makers%20must%20face." onclick="dataLayer.push({' event' : 'GAEvent', 'eventCategory' : 'product page' , 'eventAction' : 'core - left' , 'eventLabel' : 'Webinar - Introuction to Ubuntu Core 22 ' , 'eventValue' : '1' });">Introduction to Ubuntu Core 22&nbsp;&rsaquo;</a></h3>
```

Comment:
I don't think that URL chunk should be there unless it's some kind of GA query. But I've already asked Edoardo in the copy doc

Reply:
Nice, thanks @carkod !